ui/apps/platform/src/routePaths.js

```diff
@@ -23,8 +23,11 @@ export const integrationsListPath = `${integrationsPath}/:source/:type`;
 export const integrationCreatePath = `${integrationsPath}/:source/:type/create`;
 export const integrationDetailsPath = `${integrationsPath}/:source/:type/view/:id`;
 export const integrationEditPath = `${integrationsPath}/:source/:type/edit/:id`;
-export const policiesBasePath = `${mainPath}/policies`;
+export const policyManagementBasePath = `${mainPath}/policymanagement`;
```

Comment:
nit: use hyphens in URLs for spaces

```suggestion
export const policyManagementBasePath = `${mainPath}/policy-management`;
```

Reference:
https://developers.google.com/search/docs/advanced/guidelines/url-structure